libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/serial_api.c

```diff
@@ -512,7 +512,11 @@ int serial_getc(serial_t *obj) {
     int data;
     int was_masked;
 
+#if defined ( __ICCARM__ )
+    was_masked = __disable_irq_iar();
```

Comment:
But if not defined __ICCARM__, then was_masked is not valid? As __disable_irq() return type is `` void ``